style.css

```diff
@@ -75,6 +92,258 @@ nav {
   color: #344563;
 }
 
+.contact-link {
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.social-media-links {
+  display: flex;
+  gap: 20px;
+  text-decoration: none;
+}
+
+.works {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 88px;
+}
```

Comment:
- [ ] According to the project requirement the main container of the projects should have a `display: grid`. Please do so